app/src/main/res/layout/fragment_profile_new.xml

```diff
@@ -237,24 +236,19 @@
                         android:paddingLeft="32dp"
                         android:paddingTop="4dp"
                         android:textSize="@dimen/text_header"
+                        android:visibility="gone"
                         tools:ignore="MissingPrefix"
                         tools:text="@string/user_info"
                         tools:visibility="visible"/>
-
-                    <TextView
+                    
+                    <org.stepic.droid.ui.custom.LatexSupportableEnhancedFrameLayout
                         android:id="@+id/shortBioSecondText"
-                        fontPath="fonts/Roboto-Light.ttf"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
-                        android:lineSpacingExtra="3.3sp"
                         android:paddingBottom="@dimen/guideline_standard_padding"
                         android:paddingLeft="32dp"
                         android:paddingRight="32dp"
-                        android:paddingTop="4dp"
-                        android:textSize="14sp"
```

Comment:
We should set 14sp to the new view too. For example, we can create custom attr for `LatexSupportableEnhancedFrameLayout`

Reply:
It is set in `ProfileFragment#onViewCreated`